src/pages/About.vue

```diff
@@ -29,6 +146,38 @@ export default {
   },
   components: {
     History
+  },
+  data() {
+    return {
+      svc_img: "/assets/img/cloud.jpg"
+    };
+  },
+  methods: {
+    mouseOverChangeImage(i_clicked, newSrc) {
+      //console.log(elem_clicked.target)
+      var all_cards = document.getElementsByClassName(
+        "button card box is-white is-medium"
+      );
+      var active_card = document.getElementsByClassName(
+        "button card box is-white is-medium active-card"
+      );
+      for (var i = 0; i < active_card.length; i++) {
+        active_card[i].classList.remove("active-card");
+      }
+      all_cards[i_clicked].classList.add("active-card");
+      this.svc_img = newSrc;
```

Comment:
You should use ES6 here (`let` instead of `var`). You could also put a query selector after selecting all_cards:
```javascript
let all_cards = document.getElementsByClassName('button card box is-white is-medium');
let active_card = all_cards.getElementsByClassName('active-card');
```

Reply:
I'd also suggest using camelCase instead of snakeCase when dealing with JavaScript (since the native methods/functions use camelCase).